src/com/tw/go/plugin/nuget/NuGetFeedDocument.java

```diff
@@ -30,7 +31,12 @@ Date getPublishedDate() {
     }
 
     private String getProperty(NodeList properties, String name) {
-        return firstOf(properties).getElementsByTagNameNS(SCHEMA_ADO_DATASERVICES, name).item(0).getTextContent();
```

Comment:
In this case, I couldn't find a really good reason to justify returning null, hence the empty string. I will make a commit shortly with the terniary operator.